sdk/resourcemanager/Azure.ResourceManager/eng/autorest-resources.md

```diff
@@ -23,8 +23,6 @@ namespace: Azure.ResourceManager.Resources
 model-namespace: false
 public-clients: false
 head-as-boolean: false
-modelerfour:
-    lenient-model-deduplication: true
```

Comment:
As far as I know, this file does not have any effect, because the resourcemanager is generated from this configuration file: https://github.com/Azure/azure-sdk-for-net/blob/25c9dd2fb0f48628f531e16612b8cc0b0d781179/sdk/resourcemanager/Azure.ResourceManager/src/autorest.md
I am not sure why @fengzhou-msft still kept these files, but I suggest we remove this entire directory `sdk/resourcemanager/Azure.ResourceManager/eng`

Reply:
Leave it as is now